tip-wlan/charts/opensync-gw-cloud/values.yaml

```diff
@@ -166,17 +166,13 @@ ingress:
   #    hosts:
   #      - chart-example.local
 
-resources: {}
-  # We usually recommend not to specify default resources and to leave this as a conscious
-  # choice for the user. This also increases chances charts run on environments with little
-  # resources, such as Minikube. If you do want to specify resources, uncomment the following
-  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
-  # limits:
-  #   cpu: 100m
-  #   memory: 128Mi
-  # requests:
-  #   cpu: 100m
-  #   memory: 128Mi
+resources:
+  limits:
+    cpu: 500m
+    memory: 750Mi
```

Comment:
I am sure that opensync-gw will hit this limit soon'ish. I remember @cbuschfb sent a message in the wlan-cloud-platform regarding running with many Simulated APs (see here: https://telecominfraproject.slack.com/archives/G019WQETNG3/p1607697089088200). Perhaps would be good to know the limits Opensync-GW pod reached during that exercise and then using it to benchmark OpensyncGW pod limits (based on the number of APs one of the instances can manage).

Reply:
My goal with these default values is to allow Kubernetes to properly schedule a Cloud SDK deployment + a certain (we have to find out what certain means in our case) amount of additional capacity for some load. With enough APs generating load you'll always be able to exceed these limits. That's why they should be overwritten when it's necessary. Hope that makes sense.

Reply:
Yes sure. I was thinking a bit in the future I suppose, on figuring out the # of supported APs/Opensync-gw-pod before the limits set here are reached.

Reply:
Feel free to make other suggestions. I'm just missing the data to make better predictions for these values.

Reply:
Sure. I think we can continue with these for now and then update them to determine the right #s based on # of APs after experimental tests.